src/internal/client.ts

```diff
@@ -887,4 +887,14 @@ export class TypedClient {
     const xmlResult = await readAsString(httpRes)
     return xmlParsers.parseListBucket(xmlResult)
   }
+
+  removeBucketReplication(bucketName: string, callback: NoResultCallback): void
```

Comment:
missing async overload. The impl is not emit in type declaration

Reply:
Thank you so much for the valuable feedback :+1: